### PR TITLE
docs: Remove Katacoda from docs. Fixes #8840

### LINKED ISSUE #8840: Remove Katacoda links from docs by 15th June
Katacoda is being shuttered:

https://www.oreilly.com/online-learning/leveraging-katacoda-technology.html

We need to remove it from the docs:

https://github.com/argoproj/argo-workflows/search?q=katacoda

### PR DESCRIPTION
**Pull request by caelan-io**
Fixes #8840 

Removes links to the Katacoda course since Katacoda is being shuttered. Leaves references to an online course while we finish transitioning the content over to a new platform (tracked in issue #8899 ).

### CODE REVIEW

**Review by terrytangyuan — APPROVED** [review]
Thanks!

**Comment by caelan-io** [comment]
@alexec thanks, I found one typo in some old copy. please let me know what other errors you see.

**Comment by alexec** [comment]
errors are in the build:

https://github.com/argoproj/argo-workflows/runs/6715796059?check_suite_focus=true

docs/training.md
[283](https://github.com/argoproj/argo-workflows/runs/6715796059?check_suite_focus=true#step:7:284)
       17 |  to be live by Q3 '22. Please upvote or comment on [this issue](ht 
[284](https://github.com/argoproj/argo-workflows/runs/6715796059?check_suite_focus=true#step:7:285)

[285](https://github.com/argoproj/argo-workflows/runs/6715796059?check_suite_focus=true#step:7:286)
    docs/quick-start.md
[286](https://github.com/argoproj/argo-workflows/runs/6715796059?check_suite_focus=true#step:7:287)
       62 | atacoda was shut down. Please upvote or comment on [this issue](ht 
[287](https://github.com/argoproj/argo-workflows/runs/6715796059?check_suite_focus=true#step:7:288)

[288](https://github.com/argoproj/argo-workflows/runs/6715796059?check_suite_focus=true#step:7:289)
>> 2 spelling errors found in 125 files
[289](https://github.com/argoproj/argo-workflows/runs/6715796059?check_suite_focus=true#step:7:290)
make: *** [Makefile:612: docs-spellcheck] Error 1

File: docs/training.md [hunk 1]
```diff
@@ -1,17 +1,21 @@
...
+We are working on a new web-based course featuring beginner and intermediate lessons. These allow to you try out Argo Workflows in your web browser without needing to install anything on your computer. Each lesson starts up a Kubernetes cluster that you can access via a web browser.
 
-The course will take around 2 hours to complete and is the fastest way to learn Argo Workflows.
+The course takes around 2 hours to complete and is the fastest way to learn Argo Workflows.
 
-[![Katacoda Screenshot](assets/katacoda.png) Open the course](https://www.katacoda.com/argoproj/courses/argo-workflows/)
+![Katacoda Screenshot](assets/katacoda.png)
 
-## Videos
+We expect the new course to be live by Q3 '22. Please upvote or comment on [this issue](https://github.com/argoproj/argo-workflows/issues/8899) with your support and feedback for the hands-on course.
```
Comment (alexec) [inline docs/training.md:17]: lets not make any promises eh?

Comment (caelan-io) [reply docs/training.md:17]: ha, I'm pretty confident I can make it happen, but yes, happy to not set expectations too high 👍

**Review by jmeridth — APPROVED** [review]

**Comment by alexec** [comment]
LGTM. You need to fix DCO check.

**Pull request merged**